addon/templates/components/nypr-brick-item.hbs

```diff
@@ -26,12 +26,12 @@
         type='blue-minion'
         state=(if (eq (string pk) audio.currentAudio.id) audio.playState)
         itemPK=item.id
-        itemTitle=item.attributes.title}}
+        itemTitle=(if item.attributes.shortTitle item.attributes.shortTitle item.attributes.title)}}
         Listen {{item.attributes.audioDurationReadable}}
       {{/listen-button}}
     {{/if}}
   </div>
   {{#link-to 'story' item.attributes.slug class="item__link--hidden"}}
-    {{item.attributes.title}}
+    {{if item.attributes.shortTitle item.attributes.shortTitle item.attributes.title}}
```

Comment:
You know what I'm actually going to do is try to rig this up so the bucket items are converted into ember data records, which will cut down on a lot of these object paths. 

Theres an open ticket for allowing events to render through here, which conflicts with the link to component. Using models in here instead of POJOs will ease the friction for completing that ticket.